tools/example-showcase/src/main.rs

```diff
@@ -173,33 +252,118 @@ fn main() {
                 }
 
                 let before = Instant::now();
+                if report_details {
+                    cmd = cmd.ignore_status();
+                }
+                let result = cmd.output();
 
-                if cmd.run().is_ok() {
+                let duration = before.elapsed();
+                println!("took {duration:?}");
```

Comment:
```suggestion
```

Is this needed? It seems like duration is output later in reports and it messes with the progress bar.